packages/agent/conf/demos/knb/babylon/oid4vci_options/knb_van_auckland.json

```diff
@@ -1,6 +1,6 @@
 {
-  "definitionId": "knb-bank",
-  "correlationId": "https://bank.demo.sphereon.com/agent",
+  "definitionId": "knb-babylon",
+  "correlationId": "http://192.168.1.200:5000",
```

Comment:
local ip

Reply:
Not only that. The whole name change is worse. Just stick to the roles for the names, not the imaginary names we created. So knb-bank is way better than knb-bavylon, which says nothing to most people

Reply:
fixed